static/locales/en_GB.yaml

```
@@ -288,6 +288,7 @@ Settings:
       Pastel Pink: Pastel pink
       Hot Pink: Hot pink
       Nordic: Nordic
+      Gruvbox: Gruvbox
```

This should state the same as https://github.com/FreeTubeApp/FreeTube/pull/4805/files#diff-4eaadcc25bfbd7ab1ba912e36d5e1bcbab346cab59546a970df7546d5fb7a145R331

```suggestion
      Gruvbox Dark: Gruvbox Dark
```